dev-tools/packaging/packages.yml

```diff
@@ -160,7 +160,7 @@ shared:
     <<: *agent_binary_spec
     extra_vars:
       from: 'centos:7'
-      user: '{{ .BeatName }}'
+      user: 'root'
```

Comment:
@michalpristas @ph @andrewkroh @urso Just stumbled over this change. I wonder if this could become a problem or is ok?

Reply:
this one was done on purpose as agent needs root permissions, the specification here is used for agent only so no beat is affected